website/app-templates/smarty/elements/move_as_list.tpl

```diff
@@ -16,5 +16,5 @@
         <br />
         <small>{$move->author|userlink:$move->username nofilter}</small>
     </td>
-    <td class="text-right">{if $move->logtype->isCountingKilometers()}{$move->distance} km{/if}</td>
+    <td class="text-right">{if $move->logtype}{if $move->logtype->isCountingKilometers()}{$move->distance} km{/if}{/if}</td>
```

Comment:
One if is sufficient. `{If test && test->func()}{/if}`

Reply:
yep